ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_parallel_operators.py

```diff
@@ -0,0 +1,41 @@
+# global
+import ivy
+import numpy as np
+from hypothesis import assume, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# axis_index
+@handle_frontend_test(
+    fn_tree="jax.lax.axis_index",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+)
+def test_jax_lax_axis_index(
+    *,
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        axis_name="j",
+    )
```

Comment:
You are not using `x` anywhere... 
I'm not familiar with this jax function, but having a brief look at the docs it seems to me that `axis_name` has to operate on some input array. I suggest you read the documentation more carefully to figure this out.

Reply:
Thank you, I fixed some things. Can you please assist with another thing, please? My vs-code (connect to my docker), is running the code nicely, but the major issue is that I'm not able to debug the tests live with stdout, indicating what are my errors. I guess that I'm missing out on some env configuration, but couldn't find the solution online/ivy documentation.

Thank you,
Lior Tabibi

Reply:
Could you please ask this on the # vs-code forum on discord?